passage/src/main/java/id/passage/android/PassageHosted.kt

```diff
@@ -27,12 +28,19 @@ internal class PassageHosted {
         private val appId = Passage.appId
         private var packageName = ""
 
+        private fun openCustomTab(
+            activity: Activity,
+            url: String,
+        ) {
+            WebViewUtils.openCustomTab(activity, url)
+        }
+
```

Comment:
Do we need to keep this method?
```suggestion
```

Reply:
yes, we're using this private function inside `openChromeTab` and  `logout`

Reply:
Can those functions not just call `WebViewUtils.openCustomTab(activity, url)` directly?

Reply:
I kept the private function to ensure consistency. This way, both logout and openChromeTab use the same method to launch the WebView. If we need to make changes in the future, having a single function ensures everything continues to work correctly. If one of them calls the URL directly without using WebViewUtils, it might work now but could cause issues later. The goal is to make sure both are always using the same function. However, I'm open to changing it if you think it's not necessary.